Documentation/platforms/sim/sim/index.rst

```diff
@@ -103,6 +103,21 @@ You should see some squares in different colors displayed in remmina:
 
    remmina connected to sim's VNC Server
 
+Running Simulated CAN
+=====================
+
+The simulator supports CAN support via SocketCAN on the host.
+The CAN interface of the host must be properly configured::
+
+  ip link set can0 type can bitrate 1000000
+  ip link set can0 up
+
+Virtual CAN interface can be used as well::
+
+  ip link add dev can0 type vcan
+  ifconfig can0 up
+
+
```

Comment:
Please include the detailed information that you added to the Summary, showing the host side, nsh side and the expected result.

Reply:
done